web-local/tests/olap-connectors.spec.ts

```diff
@@ -0,0 +1,137 @@
+import { expect } from "@playwright/test";
+import { test } from "./setup/base";
+
+/**
+ * These tests focus on form validation and YAML generation for OLAP connectors.
+ * They don't test connections to live OLAP databases - that's handled by runtime integration tests.
```

Comment:
Do we need E2E if we dont actually test the connection? We could have component tests instead?

Reply:
I added the connection test in this PR: https://github.com/rilldata/rill/pull/6892